libraries/rush-lib/assets/rush-init/common/config/rush/subspaces.json

```diff
@@ -11,6 +11,11 @@
    */
   "subspacesEnabled": false,
 
+  /**
+   * the starting assets folder relative to rush.json location you want to use when init a subspace
+   */
+  // "subspaceInitAssetsFolder": "",
```

Comment:
In this design, it seems that a given monorepo can have at most one set of custom file templates to use with `rush init-subspace`.  Therefore, why do we need to configure its location? 

We could instead designate a standard location, e.g. `common/config-templates/rush-init-subspace`. Then everyone will know that is where to find it, without having to read **subspaces.json** to find the particular location for a particular repo.

Reply:
Or maybe `common/templates/config`